app/src/main/java/nerd/tuxmobil/fahrplan/congress/net/ConnectivityStateReceiver.java

```diff
@@ -36,27 +44,24 @@ public void onReceive(Context context, Intent intent) {
         }
     }
 
-    public static void disableReceiver(Context ctx) {
-        final PackageManager pm;
-        pm = ctx.getPackageManager();
-        ComponentName receiver = new ComponentName(ctx, ConnectivityStateReceiver.class);
-        pm.setComponentEnabledSetting(receiver, PackageManager.COMPONENT_ENABLED_STATE_DISABLED,
+    public static void disableReceiver(Context context) {
```

Comment:
It's unnecessary to enable/disable the receiver component if you dynamically register it.

Reply:
That might be true. However I don't know about the initial motivation to add this here. Therefore, I left the code as is.

Reply:
The intention was to be more energy-efficient. Disable the broadcast receiver if you're not interested in network change events. Because if you don't the app might get started to handle the broadcast even though there's no work to do. This is no longer an issue if you register the receiver dynamically.

Reply:
This sounds reasonable. Do you want to prepare a PR yourself to remove this mechanism?

Reply:
@cketti I prepared a PR here: https://github.com/EventFahrplan/EventFahrplan/pull/76 - is it ready to merge?